xbmc/dialogs/GUIDialogYesNo.h

```diff
@@ -90,6 +90,27 @@ class CGUIDialogYesNo :
    */
   static bool ShowAndGetInput(CVariant heading, CVariant text, bool &bCanceled, CVariant noLabel, CVariant yesLabel, unsigned int autoCloseTime);
 
+  /*!
+    \brief Open a Yes/No dialog and wait for input
+
+    CVariant options{CVariant::VariantTypeObject};
+    options["heading"] = int or string;
+    options["text"] = int or string;
+    options["line0"] = int or string;
+    options["line1"] = int or string;
+    options["line2"] = int or string;
+    options["yeslabel"] = int or string;
+    options["nolabel"] = int or string;
+    options["autoclose"] = int
+
+    \param[in] options  CVariant map of options described above.
+                        To avoid spelling errors it's advised to use
+                        the static strings in KODI::MESSAGING::HELPERS
+
+    \returns -1 for cancelled, 0 for No and 1 for Yes
```

Comment:
Or better: bool, or if cancelled is really needed: enum. Returning int here makes no sense